docs/architecture/adr-047-handling-evidence-from-light-client.md

```diff
@@ -0,0 +1,165 @@
+# ADR 047: Handling evidence from light client
+
+## Changelog
+* 18-02-2020: Initial draft
+* 24-02-2020: Second version
+
+## Context
+
+If the light client is under attack (either directly - lunatic / phantom
+validators or indirectly - fork on the main chain), it's supposed to halt and
+send an evidence of misbehavior to the correct full node. Upon receiving an
```

Comment:
```suggestion
send evidence of misbehavior to a full node. Upon receiving an
```
? Not sure what correct full node means, maybe I'm missing some context

Reply:
correct - not malicious